browser/net/BUILD.gn

```diff
@@ -31,23 +31,15 @@ source_set("net") {
 
   deps = [
     "//base",
+    "//brave/app:brave_generated_resources_grit",
+    "//brave/app/android:common_extensions",
```

Comment:
this has to remain in an `is_android` guard or there will be duplicate symbol errors

Reply:
actually this should be 
```if (is_android) {
  deps += [ "//brave/app/android:common_extensions" ]
} else {
  deps += [ "//extensions/common" ]
}